react2-week1/app/src/epic/page.jsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+
+import { useEffect, useState } from "react";
+
+const apiKey = "TaA8BgllKVYPCxHa4t6SkF5NemO6QmTGv4lwvRkM";
+
+export default function SearchEpicImage() {
+    const [date, setDate] = useState(""); 
```

Comment:
Nice working solution, but the requirement was to get date from query parameters (link like this: `http://localhost:3000/src/epic?search=2022-11-11`)

For that we can use `useSearchParams` hook, which returns special object with all the query parameters: https://nextjs.org/docs/app/api-reference/functions/use-search-params